src/scripts/menubuttonpage.js

```diff
@@ -0,0 +1,93 @@
+function menuButtonPage() {
+  const menuButton = document.querySelector('.menu');
+  menuButton.addEventListener('click', () => {
+    const content = document.querySelector('.content');
+    const contactContent = document.querySelector('.contact_content');
+    if (content !== null) {
+      content.parentNode.removeChild(content);
```

Comment:
- [ ] Please change this and **all** other regular functions to es6 arrow function syntax.